tools/src/test/java/org/cyclonedx/schema/JsonSchemaVerificationTest.java

```diff
@@ -33,15 +34,17 @@ Collection<DynamicTest> dynamicTestsWithCollection() throws Exception {
         final List<DynamicTest> dynamicTests = new ArrayList<>();
         for (final String file: files) {
             if (file.endsWith(".json")) {
-                final CycloneDxSchema.Version schemaVersion;
+                final Version schemaVersion;
                 if (file.endsWith("-1.2.json")) {
-                    schemaVersion = CycloneDxSchema.Version.VERSION_12;
+                    schemaVersion = Version.VERSION_12;
                 } else if (file.endsWith("-1.3.json")) {
-                    schemaVersion = CycloneDxSchema.Version.VERSION_13;
+                    schemaVersion = Version.VERSION_13;
                 } else if (file.endsWith("-1.4.json")) {
-                    schemaVersion = CycloneDxSchema.Version.VERSION_14;
+                    schemaVersion = Version.VERSION_14;
                 } else if (file.endsWith("-1.5.json")) {
-                    schemaVersion = CycloneDxSchema.Version.VERSION_15;
+                    schemaVersion = Version.VERSION_15;
+                } else if (file.endsWith("-1.6.json")) {
```

Comment:
this file seams to be slipped in by accident. it is irrelevant for the java upgrade.
please remove it and create an extra PR for it

Reply:
In this case, I think you are wrong: upgrading from [`cyclonedx-core-java-8.0.3`](https://github.com/CycloneDX/cyclonedx-core-java/releases/tag/cyclonedx-core-java-8.0.3) to [`cyclonedx-core-java-9.0.2`](https://github.com/CycloneDX/cyclonedx-core-java/releases/tag/cyclonedx-core-java-9.0.2) has an impact on the class `CycloneDxSchema`: some of its content has been moved to a new file.

The `public enum Version` has moved from file `src/main/java/org/cyclonedx/CycloneDxSchema.java` in v8.0.3 to file `src/main/java/org/cyclonedx/Version.java` in v9.0.2.

For **cyclonedx-core-java-9.0.2**: there is a new file `Version.java`: 
https://github.com/CycloneDX/cyclonedx-core-java/blob/cyclonedx-core-java-9.0.2/src/main/java/org/cyclonedx/Version.java

If you only modify the `pom.xml` to upgrade from 8.0.3 to 9.0.2, then the Java tests will fail, as they need the content from the new file `src/main/java/org/cyclonedx/Version.java` that you get from `import org.cyclonedx.Version;`.

If you want me to do an extra PR for this, please elaborate because I don't understand how to do this.

Reply:
Okay sorry I misunderstood, I though you were talking about the whole file.

But you understand that the 2 are intricate: if I do a third PR for 1.6 before you propagate the `cyclonedx-core-java` upgrade, this means I need to do extra work.

So I will remove everything related to 1.6, and I will wait for this PR about the java lib update to be merged before doing a new PR for adding 1.6 tests.